lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -433,8 +436,19 @@ UsdTransform3dMatrixOpHandler::editTransform3d(const Ufe::SceneItem::Ptr& item)
     });
     bool foundMatrix = (i != xformOps.end());
 
-    return foundMatrix ? UsdTransform3dMatrixOp::create(usdItem, *i)
-                       : _nextHandler->editTransform3d(item);
+    // If we've found a matrix op, but there is a more local non-matrix op in
```

Comment:
This is a crucial change: if there is a more local non-matrix op, let the next handler in the chain handle the edit.  This is because when a Maya fallback stack is appended to the transform ops, it must be the stack on which operations are performed.  For example, if a fallback pivot transform op is added, any further rotation on the prim must be a fallback rotation, on which the fallback pivot will apply.